plugins/pulp_rpm/plugins/importers/yum/utils.py

```diff
@@ -105,67 +105,52 @@ def strip_ns(element, uri=None):
 class RepoURLModifier(object):
     """
     Repository URL Modifier
-
-    :ivar conf: URL modifier persistent configuration
-    :type conf: dict
-
     """
-    def __init__(self, path_append=None, ensure_trailing_slash=None, query_auth_token=None):
+    def __init__(self, query_auth_token=None):
         """
-        :ivar conf: URL modifier persistent configuration, populated from optional keyword
-                    arguments (see :py:meth:`RepoURLModifier.__call__` for allowed options).
-        :type conf: dict
+        Initialize the URL modifier with defaults, populated from optional keyword arguments
```

Comment:
I did struggled a bit with wording. "from an optional keyword argument", "with a default query auth token", just end the line at "with defaults", etc. I think this is still my preferred wording.